docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -11,7 +11,15 @@ menu:
 type: docs
 ---
 
-Do the following configuration steps on each of the nodes in the cluster.
+Perform the following configuration on each node in the cluster:
+
+- ntp and chrony
+- ulimits
+- transparent hugepages
+
+Keep in mind that, although YugabyteDB is PostgreSQL compatible and runs a postgres process, it is not a PostgreSQL distribution. The PostgreSQL it runs doesn't need the same OS and system resources that open source PostgreSQL requires. For this reason, the kernel configuration requirements are different.
+
+In particular, the main YugabyteDB process, the YB-TServer, is multi-threaded. As a result, you don't need to modify settings for shared memory and inter-process communication (IPC), because there is no inter-process communication or shared memory in a multi-threaded process model (all memory is shared by the same process).
```

Comment:
This is true for yb-tserver. But each YSQL connection is still a separate postgresql process. 

The yb-tserver <---> postgres-process-per-ysql-connection is done through RPC or IPC (don't know if we've upgraded to IPC+shared_memory yet). You should ask someone from YSQL-team.

Reply:
Agree.